pkg/driver/options.go

```diff
@@ -54,6 +54,7 @@ type Options struct {
 	ExtraTags map[string]string
 	// ExtraVolumeTags is a map of tags that will be attached to each dynamically provisioned
 	// volume.
+	//
```

Comment:
New linter requires `Deprecated` comments are in their own paragraph, thus this change